test/evergreen.yml

```diff
@@ -1755,6 +1755,7 @@ buildvariants:
     smp_command: -j $(grep -c ^processor /proc/cpuinfo)
     posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-python --enable-zlib --enable-snappy --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
     make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
```

Comment:
Is it the same to the default value?

Reply:
"test_env_vars" does not have a concept of default value. It is either set or not set. The unit-test jobs of Ubuntu ppc and zseries were failing because LD_LIBRARY_PATH environment variable was not set.